From 85adede3939dca739b48f7fa98f414e4c3dd8484 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 1 Nov 2023 10:27:28 +0000
Subject: [PATCH 0696/1085] dts: bcm2712-rpi-5-b: Set enable-mps-rcb for RP1

It is safe to set the MPS_RCB bit for RP1, but not necessarily for other
devices, so add the new enabling property to pcie2 on Pi 5.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
@@ -160,6 +160,7 @@
 };
 
 rp1_target: &pcie2 {
+	brcm,enable-mps-rcb;
 	brcm,vdm-qos-map = <0xbbaa9888>;
 	aspm-no-l0s;
 	status = "okay";
